### DISCUSSION TITLE
Find the odd one

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#680] [Galaxy Trivia] [answered]
**RIDWHATW**
**ECDEER**
**CANEVAD**
**KHIRNS**

**Comment by peterbaccaro** [comment]
WITHDRAW
RECEDE
ADVANCE
SHRINK

The odd one out is **ADVANCE**.

**Reply by ParanoidUser** [reply]
Any particular reason for choosing this one? 🤓

**Reply by peterbaccaro** [reply]
WITHDRAW, RECEDE and SHRINK all have a similar meaning to each other. 
ADVANCE on the other hand has the opposite meaning of those words.

**Comment by thuongtruong1009** [comment, marked as answer]
The odd one out is "KHIRNS".

All of the other options can be rearranged to form valid English words:

"RIDWHATW" can be rearranged to form "DRAWTHIR".
"ECDEER" can be rearranged to form "DECREE".
"CANEVAD" can be rearranged to form "ADVANCE".
However, "KHIRNS" cannot be rearranged to form a valid English word
